sphinx/ext/auto_cli_doc.py

```diff
@@ -328,9 +328,6 @@ def write_command_reference(write, commands):
             (a.strip(), b) for a, b in
             ROSE_HELP_SECTION_REGEX.findall(help_text))
 
-        # The NAME section is not used.
-        del sections['NAME']
```

Comment:
I think this should stay. This line enforces that command usage includes a `NAME` section.